test/workflows/environments/kubeflow/spec.json

```diff
@@ -0,0 +1,4 @@
+{
+  "server": "https://35.196.185.88",
```

Comment:
Where is this value being used?

Reply:
This is a ksonnet application that allows you to run the workflows. This is an environment that points at the cluster we use for E2E testing.